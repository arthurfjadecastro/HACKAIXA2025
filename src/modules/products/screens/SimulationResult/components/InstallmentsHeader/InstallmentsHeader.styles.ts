import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
  installmentsHeader: {
    paddingHorizontal: 0,
    paddingTop: 32,
    paddingBottom: 16,
  },

  installmentsTitle: {
    fontSize: 18,
    fontWeight: '800',
    color: '#1565C0',
    textAlign: 'left',
  },
});
