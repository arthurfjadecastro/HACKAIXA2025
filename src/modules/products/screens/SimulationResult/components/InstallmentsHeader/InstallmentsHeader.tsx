import React from 'react';
import { View } from 'react-native';
import { Text } from '@/design-system/components';
import { styles } from './InstallmentsHeader.styles';

export const InstallmentsHeader: React.FC = () => {
  return (
    <View style={styles.installmentsHeader}>
      <Text style={styles.installmentsTitle}>Parcelas</Text>
    </View>
  );
};
