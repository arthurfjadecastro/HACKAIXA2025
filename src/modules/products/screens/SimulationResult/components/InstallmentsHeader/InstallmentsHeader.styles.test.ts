import { styles } from './InstallmentsHeader.styles';

describe('InstallmentsHeader Styles', () => {
  it('should export styles object', () => {
    expect(styles).toBeDefined();
    expect(typeof styles).toBe('object');
  });

  it('should have installmentsHeader style', () => {
    expect(styles.installmentsHeader).toBeDefined();
    expect(styles.installmentsHeader.paddingHorizontal).toBe(0);
    expect(styles.installmentsHeader.paddingTop).toBe(32);
    expect(styles.installmentsHeader.paddingBottom).toBe(16);
  });

  it('should have installmentsTitle style', () => {
    expect(styles.installmentsTitle).toBeDefined();
    expect(styles.installmentsTitle.fontSize).toBe(18);
    expect(styles.installmentsTitle.fontWeight).toBe('800');
    expect(styles.installmentsTitle.color).toBe('#1565C0');
    expect(styles.installmentsTitle.textAlign).toBe('left');
  });

  it('should match snapshot', () => {
    expect(styles).toMatchSnapshot();
  });
});
