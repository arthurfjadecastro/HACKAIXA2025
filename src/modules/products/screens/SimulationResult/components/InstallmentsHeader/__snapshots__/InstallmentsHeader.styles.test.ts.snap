// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallmentsHeader Styles should match snapshot 1`] = `
{
  "installmentsHeader": {
    "paddingBottom": 16,
    "paddingHorizontal": 0,
    "paddingTop": 32,
  },
  "installmentsTitle": {
    "color": "#1565C0",
    "fontSize": 18,
    "fontWeight": "800",
    "textAlign": "left",
  },
}
`;
